### PR TITLE
Add twitch context for email generation, PLATFORM-4099

### PR DESCRIPTION
**Pull request by abador**

### CODE REVIEW

File: extensions/wikia/Email/i18n/TwitchDisconnect.i18n.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+$messages['ja'] = array(
+	'emailext-twitchdisconnect-subject' => 'FANDOMアカウントとTwitchとの連携が解除されました',
+	'emailext-twitchdisconnect-summary' => 'FANDOMアカウントとTwitchアカウントの連携が解除されました。FANDOMコミュニティ上での履歴・メッセージ・プロフィールなど、大切な情報はすべて保持されていますので、ご安心ください。',
+	'emailext-twitchdisconnect-intro' => '同じユーザー名を継続して使用する場合、以下のリンクから新しいパスワードを設定する必要があります。',
+	'emailext-twitchdisconnect-instructions' => '該当のアカウントでパスワードを設定している場合、再設定の必要はありません。引き続き設定済みのパスワードでFANDOMをご利用いただけます。パスワードの設定を行っていない場合は、お手数ですが上記のリンクからパスワードを設定してください。',
+);
+
+$messages['ko'] = array(
+	'emailext-twitchdisconnect-instructions' => '이미 위키아 비밀번호를 설정하셨다면 걱정하지 마세요! 귀하의 계정은 안전합니다. 이 이메일은 무시하시고 기존의 비밀번호로 로그인하시면 됩니다. 그 외의 경우에는 임시 비밀번호로 로그인한 뒤 환경 설정에서 원하는 비밀번호로 설정을 변경하시면 됩니다.',
+	'emailext-twitchdisconnect-subject' => '위키아 계정의 페이스북 연동이 해제되었습니다',
```
Comment (themech) [inline extensions/wikia/Email/i18n/TwitchDisconnect.i18n.php]: Google Translate claims this means `The Facebook link of the Wikia account has been canceled`. Shouldn't we just wait for translations from crowdin?

File: extensions/wikia/Email/i18n/TwitchDisconnect.i18n.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<?php
```
Comment (themech) [inline extensions/wikia/Email/i18n/TwitchDisconnect.i18n.php:1]: I think you should create `crowdin-twitchdisconnect.conf` file (based on configs we have for FB and Google).

**Review by themech — CHANGES_REQUESTED** [review]

File: extensions/wikia/Email/i18n/TwitchDisconnect.i18n.php (hunk 1 above)
Comment (abador) [reply extensions/wikia/Email/i18n/TwitchDisconnect.i18n.php]: Sorry my Korean is not too good... ;P

File: includes/wikia/controllers/HeliosHelperController.class.php [hunk 3]
```diff
@@ -118,8 +118,11 @@ private function emailControllerForTokenContext( string $tokenContext = null ):
 		if ( $tokenContext === 'facebook' ) {
 		    return 'Email\Controller\FacebookDisconnect';
         }
-        if ( $tokenContext === 'google' ) {
-		    return 'Email\Controller\GoogleDisconnect';
+		if ( $tokenContext === 'google' ) {
+			return 'Email\Controller\GoogleDisconnect';
+		}
+		if ( $tokenContext === 'twitch' ) {
+			return 'Email\Controller\GoogleDisconnect';
```
Comment (sau226) [inline includes/wikia/controllers/HeliosHelperController.class.php]: Shouldn't this read TwitchDisconnect?

```suggestion
			return 'Email\Controller\TwitchDisconnect';
```

**Review by sau226 — COMMENTED** [review]
Not a Wikia employee (just a quick review)

**Review by themech — APPROVED** [review]

**Pull request merged**